custom-elements/registries/Element-innerHTML.html

```diff
@@ -32,13 +32,46 @@
     class ScopedOtherElement extends HTMLElement { };
     registry.define('other-element', ScopedOtherElement);
 
-    const shadowRoot = createConnectedShadowTree(test, registry);
     const someElement = document.createElement('some-element', {customElementRegistry: registry});
     assert_true(someElement instanceof ScopedSomeElement);
     someElement.innerHTML = '<other-element></other-element>';
     assert_true(someElement.querySelector('other-element') instanceof ScopedOtherElement);
 }, 'innerHTML on a disconnected element should use the scoped registry it was created with');
 
+test((test) => {
+    const registry = new CustomElementRegistry;
+
+    class ScopedSomeElement extends HTMLElement { };
+    registry.define('some-element', ScopedSomeElement);
+
+    class ScopedOtherElement extends HTMLElement { };
+    registry.define('other-element', ScopedOtherElement);
+
+    const someElement = document.createElement('some-element', {customElementRegistry: registry});
+    assert_true(someElement instanceof ScopedSomeElement);
+    someElement.innerHTML = `
+        <other-element id="foo">
+            <other-element id="bar">
+                <div id="baz">
+                    <other-element id="nested-ce"></other-element>
+                </div>
+            </other-element
+        </other-element>
+        <div id="bux"></div>
+        <another-element></another-element>`
+
+    assert_true(someElmeents.querySelector('#foo') instanceof ScopedOtherElement);
```

Comment:
Did you attempt to run this test? Given it contains typos (`someElmeents` and `someElements` below, both should be `someElement` I think?) I'm a bit worried this hasn't been properly tested.